### PR TITLE
add user/group/subgroup elements to user sid55 and group sid tests

### PR DESCRIPTION
**Pull request by vanderpol**
Creating PR for additions to the user_sid55 and group_sid tests

### CODE REVIEW

File: oval-schemas/windows-definitions-schema.xsd [hunk 1]
```diff
@@ -3322,6 +3322,23 @@
...
                                           </xsd:annotation>
                                     </xsd:element>
+									  <xsd:element name="group" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="1">
+										   <xsd:annotation>
+												<xsd:documentation>A string the represents the name of a particular group. In Windows, group names are case-insensitive. As a result, it is recommended that the case-insensitive operations are used for this entity. In a domain environment, groups should be identified in the form: "domain\group name". For local groups use: "computer name\group name". For built-in accounts on the system, use the group name without a domain.</xsd:documentation>
+										   </xsd:annotation>
+									  </xsd:element>
+									  <xsd:element name="user" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="unbounded">
```
Comment (solind) [inline oval-schemas/windows-definitions-schema.xsd:3330]: Still has maxOccurs="unbounded", should be 1

File: oval-schemas/windows-system-characteristics-schema.xsd [hunk 2]
```diff
@@ -1412,6 +1412,23 @@
...
                                         </xsd:annotation>
                                    </xsd:element>
+								  <xsd:element name="group" type="oval-sc:EntityItemStringType" minOccurs="0" maxOccurs="1">
+									   <xsd:annotation>
+											<xsd:documentation>A string the represents the name of a particular group. In Windows, group names are case-insensitive. As a result, it is recommended that the case-insensitive operations are used for this entity. In a domain environment, groups should be identified in the form: "domain\group name". For local groups use: "computer name\group name". For built-in accounts on the system, use the group name without a domain.</xsd:documentation>
+									   </xsd:annotation>
+								  </xsd:element>
+								  <xsd:element name="user" type="oval-sc:EntityItemStringType" minOccurs="0" maxOccurs="unbounded">
```
Comment (solind) [inline oval-schemas/windows-system-characteristics-schema.xsd:1420]: ibid

File: oval-schemas/windows-system-characteristics-schema.xsd [hunk 3]
```diff
@@ -1412,6 +1412,23 @@
...
+									   <xsd:annotation>
+											<xsd:documentation>A string that represents the name of a particular user. In Windows, user names are case-insensitive. As a result, it is recommended that the case-insensitive operations are used for this entity. In a domain environment, users should be identified in the form: "domain\user name". For local users use: "computer name\user name". For built-in accounts on the system, use the user name without a domain.</xsd:documentation>
+											<xsd:documentation>If the specified group has more than one user as a member, then multiple user elements should exist. If the specified group does not contain a single user, then a single user element should exist with a status of 'does not exist'. If there is an error determining the users that are members of the group, then a single user element should be included with a status of 'error'.</xsd:documentation>
+									   </xsd:annotation>
+								  </xsd:element>
+								  <xsd:element name="subgroup" type="oval-sc:EntityItemStringType" minOccurs="0" maxOccurs="unbounded">
```
Comment (solind) [inline oval-schemas/windows-system-characteristics-schema.xsd:1426]: ibid

**Review by solind — CHANGES_REQUESTED** [review]
See comments; the new state entities should have maxOccurs="1" (not unbounded)

**Comment by vanderpol** [comment]
Sorry about that.. I thought I had those all removed.   Should be 'done' now.

**Review by solind — APPROVED** [review]

**Pull request merged**